src/08_ecosystem/00_chapter.md

```diff
@@ -4,6 +4,10 @@ Importantly, executors, tasks, reactors, combinators, and low-level I/O futures
 are not yet provided in the standard library. In the meantime,
 community-provided async ecosystems fill in these gaps.
 
+The Async Foundations Team is interested in extending examples in the Async Book to cover multiple runtimes.
+If you're interested in contributing to this project, please reach out to us on
+[Zulip](https://rust-lang.zulipchat.com/#narrow/stream/187312-wg-async-foundations).
```

Comment:
We might want to boost the async book topic: https://rust-lang.zulipchat.com/#narrow/stream/201246-wg-async-foundations.2Fbook

I just got wind of this recently, but seems appropriate to delegate future discussions there.